test/integration/smoke/test_list_ids_parameter.py

```diff
@@ -194,10 +194,7 @@ def setUpClass(cls):
         cls._cleanup = [
                         cls.disk_offering,
                         cls.account,
-                        cls.service_offering,
-                        cls.snapshot_1,
-                        cls.snapshot_2,
-                        cls.snapshot_3
+                        cls.service_offering
```

Comment:
I would say that cleanup should happen in reverse order of creation, irrespective of ownership. Leaving these items after an integration test while other tests may follow seems like a bad idea.